salt/metalk8s/addons/logging/deployed/dashboards.sls

```diff
@@ -0,0 +1,21 @@
+#!jinja | metalk8s_kubernetes
+{%- from "metalk8s/addons/prometheus-operator/macros.j2"
+    import configmaps_from_dashboards with context %}
+
+{{ configmaps_from_dashboards([
+      {
+          'name': 'fluent-bit',
+          'title': 'Fluent Bit',
+          'tags': ['logging'],
+      },
+      {
+          'name': 'logs',
+          'title': 'Logs',
+          'tags': ['logging'],
+      },
+      {
+          'name': 'loki',
+          'title': 'Loki',
+          'tags': ['logging'],
+      },
+]) }}
```

Comment:
It was mostly to keep things simple, but I could have kept the previous separation, you're right. Just thought that we would always need to keep the full `metalk8s.addons.logging` in sync (since fluent-bit and loki need to talk to each other).